### PR TITLE
add functionality for streaming and decimate files from grane (.sgd)

### PR DESCRIPTION
**Pull request by KristianKjerstad**
Note: the changes use the Equinor python library segdpy, which must be locally installed.

### CODE REVIEW

**Comment by KristianKjerstad** [comment]
Are there any simple ways to include the [segdpy package](https://github.com/equinor/segdpy/tree/master/segdpy) with docker?

File: seis_ee/find_files/grane.py [hunk 1]
```diff
@@ -5,6 +5,13 @@
...
+def grane_path_to_date(path: str) -> datetime:
+    #assume grane file name is on the format: 2019-10-26-07-43-59-Grane1035406.sgd
+    parts = path.split("/")
+    filename = parts[-1]
+    filename_parts = filename.split("-")
+    year, month, day = filename_parts[0], filename_parts[1], filename_parts[2]
+    return datetime(year=int(year), month=int(month), day=int(day))
 
 def grane_path_to_dates(path: Path):
     # Try splitting the filename
```
Comment (soofstad) [inline seis_ee/find_files/grane.py:17]: Possible to reuse this function in the one below?

File: seis_ee/find_files/oseberg.py [hunk 2]
```diff
@@ -139,8 +139,8 @@ def requested_times_to_oseberg_paths(requested_times, target) -> [Path]:
     paths = []
     for chunk in time_chunks:
         path, file_time = datetime_to_oseberg_path(chunk["chunk"], target)
-        paths.append({"path": path, "file_time": file_time, "event": chunk["event"]})
-
+        if (path != None and file_time != None ):
```
Comment (soofstad) [inline seis_ee/find_files/oseberg.py:142]: Not sure we want this.
We want to be able to see when there are missing files in a time period

**Review by soofstad — CHANGES_REQUESTED** [review]
Nice job. Looks good :)

**Pull request merged**